salt/__init__.py

```diff
@@ -25,24 +26,63 @@ def find_module(self, module_name, package_path=None):
                 return self
         return None
 
-    def load_module(self, name):
```

Comment:
We shouldn't remove a method without first deprecating it.

Reply:
This does need to go away, it's triggering a python deprecation warning for quite some time now.

Reply:
@Ch3LL for additional info, this is not our public API, it's python's.